misc/snapshotter/entrypoint.sh

```diff
@@ -17,8 +17,8 @@ if [ "$#" -eq 0 ]; then
 		--root ${NYDUS_LIB} \
 		--address ${NYDUS_RUN}/containerd-nydus-grpc.sock \
 		--log-level ${LEVEL} \
-		--enable-metrics ${ENABLE_METRICS} \
-		--enable-nydus-overlayfs ${ENABLE_NYDUS_OVERLAY} \
+		--enable-metrics=${ENABLE_METRICS} \
+		--enable-nydus-overlayfs=${ENABLE_NYDUS_OVERLAY} \
```

Comment:
Do we need to remove this too? Otherwise, the executive `nydus-overlayfs` should be installed

Reply:
I see. It is false, please ignore it.